tests/cypress/tests/e2e/admin.spec.js

```diff
@@ -0,0 +1,67 @@
+// element selectors
+const menuButton = '[data-testid="header-menu-dropdown-button"]';
+const menuOptionManageAccount =
+  '[data-testid="header-menu-option-manage-account"]';
+const adminButton = '[data-testid="banner-admin-button"]';
+// text selectors
+const bannerFetchErrorMessage =
+  "Banner could not be fetched. Please contact support.";
+const bannerWriteErrorMessage =
+  "Current banner could not be replaced. Please contact support.";
+const bannerDeleteErrorMessage =
+  "Current banner could not be deleted. Please contact support.";
+const noCurrentBannerMessage = "There is no current banner";
+
+beforeEach(() => {
+  cy.visit("/");
+  cy.authenticate("adminUser");
+  cy.get(menuButton).click();
+  cy.get(menuOptionManageAccount).click();
+  cy.get(adminButton).click();
+});
+
+describe("Admin banner integration tests", () => {
+  it("Fetches banner, if any, without error", () => {
+    cy.contains(bannerFetchErrorMessage).should("not.exist");
+    cy.contains(noCurrentBannerMessage).should("be.visible");
```

Comment:
Is there some common text we can test for instead? or maybe just that that element is non-empty?

Reply:
Sadly there is no common text, neither is there an easy way to test that the element is non-empty because it is technically always non-empty (because of the `<Collapse />` element). 

I looked into using some kind of `if/else` or `or` statement to test for A || B, but Cypress knew what I was after and was nice enough to have written up a [lengthy explanation why that is impossible for us](https://docs.cypress.io/guides/core-concepts/conditional-testing).

So I've made a change and here's my thinking: 

1. By moving the fetch-related test to the end we can control the state (i.e. whether there is a valid banner there to be fetched or not). In the first test we write (or overwrite, if there's already one there) a banner to the db. In the second we delete that db. Now in the third we attempt to fetch a db with an additional (and actually unnecessary) `.reload()`. We know there will (virtually) never be a banner because we just deleted it, so we can safely test for only one of the two conditions: that the "nope no banner here" text will display.
2. The `fetch` method is called after both `write` and `delete` operations, so there's some added implicit test coverage there.

Thoughts?